test/log-generators/pipelines/kaconf.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     - selector:
         apiVersion: tekton.dev/v1
         kind: TaskRun
-      archiveOnDelete: "true"
```

Comment:
Why all the changes from double to single quotes?

Maybe we should enforce something so we are consistent?

https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.quoted_strings

Reply:
I fixed the changes, now my editor works with double quote.